resources/repos.json

```
@@ -1561,5 +1561,23 @@
                 "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#31\">RC-31 or higher</a>"
             }
         }
+    },
+    "JasperChaseTOQ/CompressionCraft:master": {
+        "sonar": {
+            "enabled": false
+        },
+        "options": {
+            "prefix": "DEV",
+            "abandoned": false,
+            "createJar": true
+        },
+        "dependencies": {
+            "Minecraft Version(s)": {
+                "1": ",1.18.x,1.19x"
```

There is an unnecessary comma here
```suggestion
                "1": "1.18.x, 1.19x"
```